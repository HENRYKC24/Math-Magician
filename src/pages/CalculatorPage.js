import React from 'react';
import Calculator from '../components/Calculator';

const Calculator2 = () => (
  <div className="calculator-page">
    <p className="lets">Let&apos;s do some math!</p>
    <Calculator />
  </div>
);

export default Calculator2;
