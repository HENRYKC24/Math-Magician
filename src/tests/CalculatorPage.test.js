import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../pages/CalculatorPage';

it('Calculator page renders the UI as expected', () => {
  const tree = renderer.create(<CalculatorPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
