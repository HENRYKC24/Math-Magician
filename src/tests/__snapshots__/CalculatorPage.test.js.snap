// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator page renders the UI as expected 1`] = `
<div
  className="calculator-page"
>
  <p
    className="lets"
  >
    Let's do some math!
  </p>
  <table
    className="table"
  >
    <tbody>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div>
            <span
              className="result"
            >
              0
            </span>
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divAC"
            onClick={[Function]}
          >
            AC
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divPlusMinus"
            onClick={[Function]}
          >
            +/-
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divModulus"
            onClick={[Function]}
          >
            %
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divDivide"
            onClick={[Function]}
          >
            รท
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div7"
            onClick={[Function]}
          >
            7
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div8"
            onClick={[Function]}
          >
            8
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div9"
            onClick={[Function]}
          >
            9
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divx"
            onClick={[Function]}
          >
            x
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div4"
            onClick={[Function]}
          >
            4
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div5"
            onClick={[Function]}
          >
            5
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div6"
            onClick={[Function]}
          >
            6
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divSubtract"
            onClick={[Function]}
          >
            -
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div1"
            onClick={[Function]}
          >
            1
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div2"
            onClick={[Function]}
          >
            2
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div3"
            onClick={[Function]}
          >
            3
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divAdd"
            onClick={[Function]}
          >
            +
          </div>
        </td>
      </tr>
      <tr
        className="tr"
      >
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="div0"
            onClick={[Function]}
          >
            0
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divDot"
            onClick={[Function]}
          >
            .
          </div>
        </td>
        <td
          className="td"
        >
          <div
            aria-hidden={true}
            className="divEqual"
            onClick={[Function]}
          >
            =
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
